src/workerinfo.js

```diff
@@ -34,57 +35,65 @@ class WorkerInfo {
     }
   }
 
-  async seen(provisionerId, workerType) {
+  async seen(provisionerId, workerType, workerGroup, workerId) {
     const expired = entity => Date.now() - new Date(entity.expires) > 24 * 60 * 60 * 1000;
     const expires = taskcluster.fromNow('5 days');  // temporary hard coded expiration
 
+    const updateExpiration = (entry) => {
```

Comment:
We could also do this as a global helper... like:
```js
await workerType.modify(updateExpires);
```
or something like: 
```js
await workerType.modify(updateExpires(offset, ...));
```

On both cases `updateExpires` would be a global helper function defined inside the file and not exported...